### DISCUSSION TITLE
Is Emmetts orm also async

### DISCUSSION THREAD

**Discussion by KellerKev** [emmett-framework/emmett#334] [Q&A] [answered]
Would you mind elaborating a bit on the async features of Emmetts orm?

Is it synchronous or asynchronous?

How does it handle async safely?

Thank you!

**Comment by KellerKev** [comment, marked as answer]
To answer my own question after some research. I understand that it's not async but that is fine as db speeds these days are so fast that it's really rarely needed even with async views.

**Reply by KellerKev** [reply]
But please do correct me if I'm wrong and Emmetts orm is actually async too.

**Reply by gi0baro** [reply]
@Kkeller83 correct, the ORM is synchronous, async is targeted for 3.0 (see https://github.com/emmett-framework/emmett/issues/235)
